pkgs/tools/security/bitwarden-cli/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, nodejs }:
+
+stdenv.mkDerivation rec {
+  name = "bitwarden-cli-${version}";
+  version = "v1.6.0";
```

Comment:
Please move the "v" to the rev, keeping the version "1.6.0" :)

Reply:
I've pushed a commit to fix the version string.  I only found one instance of trailing white space and I went ahead and cleaned that up as well.